lib/elements/rh-context-picker/rh-context-picker.ts

```diff
@@ -79,44 +68,25 @@ export class RhContextPicker extends LitElement {
     },
   }) allow = RhContextPicker.paletteNames;
 
-  #offset = RhContextPicker.offsets[this.value];
-
   #internals = this.attachInternals();
 
   #target: HTMLElement | null = null;
 
-  willUpdate() {
-    this.#offset = RhContextPicker.offsets[this.value];
-  }
-
   render() {
-    const { on = 'dark', value = 'darkest' } = this;
-    const derivedLabel = this.#internals.ariaLabel
-      ?? Array.from(this.#internals.labels, x => x.textContent).join();
+    const { allow, on = 'dark', value = 'darkest' } = this;
     return html`
-      <div id="container" class="${classMap({ [on]: true })}">
-          <input id="context-range"
-                 class="${classMap({ [value]: true })}"
-                 name="range"
-                 type="range"
-                 list="palettes"
-                 min="0"
-                 max="${this.allow.length - 1}"
-                .value="${this.allow.indexOf(this.value).toString()}"
-                 aria-label="${derivedLabel}"
-                 style="${styleMap({
-                   '--count': `${this.allow.length}`,
-                   '--offset': `${this.#offset}px`,
-                  })}"
-                 @input="${this.#onInput}">
-          <datalist id="palettes">${this.allow.map(palette => html`
-            <option id="option-${palette}"
-                    value="${palette}"
-                    title="${palette}"
-                    @click="${() => this.#setValue(palette)}">
-              <span class="visually-hidden">${palette}</span>
-            </option>`)}
-          </datalist>
+      <div id="container"
```

Comment:
Probably most semantically correct to wrap these inputs with `<fieldset>` and `<legend>` tags. [WebAIM example](https://webaim.org/techniques/forms/controls#radio).

Reply:
to reiterate, it'd be 

```html
<form>
  <fieldset>
    <legend>Theming</legend>
    <label for="context">Palette</label>
    <rh-context-picker id="context"></rh-context-picker>
  </fieldset>
</form>
```
instead of

```html
<form>
  <rh-context-picker id="context">
    #shadow-root
      <fieldset>
        <legend>...</legend>
        ...
      </fieldset>
  </rh-context-picker>
</form>
```

Does that track?

Reply:
The first example is close. Here's what I suggest:

```html
<form>
  <fieldset>
    <legend>Color Palette</legend>
    <rh-context-picker></rh-context-picker>
  </fieldset>
</form>
```

👍

Reply:
see if [3d76f52](https://github.com/RedHat-UX/red-hat-design-system/pull/1581/commits/3d76f52189bb747345b4cf01daeb9ec55153343b) helps

Reply:
Here's what I would suggest:

```diff
- <label for="radio-${palette}" slot="content">${palette}</label>
+ <span slot="content" class="capitalize">${palette}</span>
+ <label class="visually-hidden capitalize" for="radio-${palette}">${palette}</label>
```

For more context, here's the [entire render method](https://pastebin.com/QTYCB2FX).

Since the previous `<label slot="content">` was being hidden by `<rh-tooltip>`, I added a `<span>` that could take its place. 

This makes AT happy. Automated tests (Lighthouse, Axe, probably Lumar) won't surface false positive accessibility errors (they were before). Safari reads:

> Dark color palette, selected, radio button 2 of 6

Which is what we want ✔️ . 

Be sure to add `.capitalize { text-transform: capitalize; }` to the CSS file too.

Reply:
These changes look good. 👏 

As seen in Office Hours, the nested context picker still has some strange click/focus issues. [Video Demo (WebM)](https://github.com/RedHat-UX/red-hat-design-system/assets/575865/afe3d3d9-7046-49e5-990d-a5b19f7be7fb)

Reply:
I'd like to spin that off into a separate issue. I suspect it's a problem with tooltip, rather than with context picker.

Repro: https://codepen.io/bennyp/pen/yLWPgew?editors=1000

Reply:
Perhaps I'm missing something, but tooltip in your repro appears to behave as expected.

Reply:
OK, yup. I see it now. It's definitely most apparent in Firefox. Thanks for clarifying.

Do you want to create and resolve that (separate) issue before resolving this conversation and merging this PR?

Reply:
https://github.com/RedHat-UX/red-hat-design-system/issues/1618